packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,336 @@
+
+#### Details card
+Recall that in Step 1, we defined the Details card as needing to display the cluster name, the provider, and the OS version. This can easily be accomplished by using a standard details card.
+
+<img src="./img/details-card-example.png" alt="detail card content" width="439"/>
+
+#### Performance card
+The performance card needs to display current usage and usage history over time for five different measures: IOPS, Latency, Throughput, Recovery rate, and Recovery IOPS. For this use case, we can group five standard trend cards and arrange them in a horizontal grid. Then, we can add a filter in the header to change the scope of the data that we wish to measure.
+
+<img src="./img/performance-card-example.png" alt="performance card content" width="1185"/>
+
+#### Health card
+The health card will be a custom card to show the state of the cluster and the cluster data. While we are not using a standard card here, we can still borrow elements of the aggregate status card for displaying overall health.
+
+<img src="./img/health-card-example.png" alt="health card content" width="670"/>
+
+#### Capacity card
+The capacity card must reflect:
+- Physical capacity (percentage used).
+- Requested capacity (percentage used).
+
+Similar to the way we grouped multiple cards within a larger card for performance, we can display these two metrics side-by-side using two donut utilization charts.
+
+<img src="./img/capacity-card-example.png" alt="capacity card content" width="735"/>
+
+#### Top consumers card
+This card gives users a glimpse of which  projects, pods, or clusters consume the most resources. We want to allow users to view this information by project, nodes, classes, and workloads for the top five consumers for any of these attributes. To accomplish this goal, we can create a custom card that uses a line chart to show consumption over time. The user can compare all five results or filter down to a specific time. We can also include an additional filter for users to change the displayed time range.
+
+<img src="./img/consumers-card-example.png" alt="top consumers card content" width="746"/>
+
+#### Events card
+The events card must display a list of system events and provide the ability to filter based on event type. For this use case, we can use a standard events card.
+
+<img src="./img/events-card-example.png" alt="events card content" width="439"/>
+
+#### Inventory card
+The inventory card is intended to list a count of current nodes, disks, pods, PVs, PVCs, and show if any are down or in error. For this use case, we can create a custom card. But since it has similar properties to the [aggregate status card](#aggregate-status-card) for displaying object status, we can pull from some of the aggregate status card’s concepts.
+
+<img src="./img/inventory-card-example.png" alt="inventory card content" width="439"/>
```

Comment:
Just realized that this image also has an info icon, and would want to switch that out for a question-circle icon. Sorry I didn't notice this earlier!

Reply:
@mmenestr I made the update. Please give it one more review.